packages/block-editor/src/hooks/anchor.js

```diff
@@ -71,7 +71,16 @@ export const withInspectorControl = createHigherOrderComponent( ( BlockEdit ) =>
 					<InspectorAdvancedControls>
 						<TextControl
 							label={ __( 'HTML Anchor' ) }
-							help={ __( 'Anchors lets you link directly to a section on a page.' ) }
+							help={ (
+							    <>
+							        { __( 'Enter a word or two — without spaces — to make a unique web address just for this heading, called an "anchor." Then, you’ll be able link directly to this section of your page.' ) }
+							        <div className="components-placeholder__learn-more">
```

Comment:
I need to change this to `<div className="components-base-control__learn-more">`